tools/startTest.sh

```diff
@@ -0,0 +1,12 @@
+#/bin/bash
+# Change to directory of script
+cd "${0%/*}/../maps/raw/"
+# Clear out old maps
+find ../ -maxdepth 1 -type f -delete
+# Get a list of files in raw
+
+# Name the file after the raw name
+for i in `ls -1 `; do
+   eval "../../tools/loadMap.py $i"
```

Comment:
On my Mac, I changed this to eval "../../tools/loadMap.py $i >../$i.html"  Having that shell redirect was necessary for me to regenerate the files in maps/*.html. I might be missing some of your changes from git, because it looks like your changes above in loadMap.py open and write to the html file.

Reply:
Those should be generated by the loadMap.py file, the script no longer should output to the shell. Hmm, I wonder if the sys redirect to file is OS specific.

Reply:
A possibly more OS agnostic way would be to use the file.write() function, but we'll also need to add newlines explicitly, where print implicitly puts one unless you specify the end character.

Reply:
I am curious about learning these differences because we are in a school environment. However, these Mac-specific issues don't bother me because I have a workaround. Everyone else (except Aiden) is using PSU's Babbage linux server, and it works for them. So, let's commit it.